Helper/NotifierResult.php

```diff
@@ -50,4 +55,15 @@ public function setResponseData($responseData)
         $this->_responseData = $responseData;
         return $this;
     }
+
+    public function getUuid(): string
+    {
+        return $this->uuid;
```

Comment:
Just to point out, this file now has two formattings of class variables in the one class, which is probably fine, but isn't cohesive.
```
$this->_subscriptionId
$this->uuid
```

Reply:
agreed, i want to move this out of the helper and use a `DataObject` instead. i will do that in a follow up code clean up PR